component/host.go

```diff
@@ -50,7 +50,7 @@ type Host interface {
 	// GetExporters can be called by the component anytime after Component.Start() begins and
 	// until Component.Shutdown() ends.
 	//
-	// Deprecated: [0.79.0] This function will be removed in the future.
+	// Deprecated: `[0.79.0]` This function will be removed in the future.
```

Comment:
Accidental change? It doesn't seem like we follow this pattern in other places

Reply:
Fixed.